tests/util/addresses.go

```diff
@@ -12,3 +13,8 @@ func SdkToEvmAddress(addr sdk.AccAddress) common.Address {
 func EvmToSdkAddress(addr common.Address) sdk.AccAddress {
 	return sdk.AccAddress(addr.Bytes())
 }
+
+func RandomAccAddress() sdk.AccAddress {
+	_, _, addr := testdata.KeyTestPubAddr()
+	return addr
+}
```

Comment:
i think we should prefer using `app.RandomAddress()` rather than add another method for this.